database/data/schemaHelpers.js

```diff
@@ -0,0 +1,43 @@
+const db = require('../index');
+
+module.exports = {
+  songSaver: (song, cb) => {
+    // TBD: write code to generate a single song instance
+    const stmt = `insert into songs (length, timestamp, isliked, songfile, title, artist, album, thumbnail) 
```

Comment:
As this is a mySQL command, following the convention of uppercase letters for keywords is preferred for ease of understanding where a keyword phrase is completed and user-defined information begins (and vice-versa).

i.e.:
```suggestion
    const stmt = `INSERT INTO songs (length, timestamp, isliked, songfile, title, artist, album, thumbnail) // etc.
```

Reply:
Good point!